docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+services:
+  runner:
+    build:
+      context: .
+      args:
+        RUBY_VERSION: ${RUBY_VERSION:-2.3.1}
+        BUNDLER_VERSION: ${BUNDLER_VERSION:-}
+    volumes:
+      - type: bind
+        source: .
+        target: /app
```

Comment:
nitpick - given volume path mount in dockerfile, might be worthwhile keeping here as well - https://docs.docker.com/reference/compose-file/services/#working_dir

duplicated but less likely for error